ocs_ci/ocs/resources/bucket_notifications_manager.py

```diff
@@ -0,0 +1,362 @@
+            name(str|optional): Name of the Kafka topic
+
+        Returns:
+            str: Name of the created Kafka topic
+        """
+        topic_name = topic_name or create_unique_resource_name(
+            resource_description="nb-notif", resource_type="kafka-topic"
+        )
+        topic = self.amq.create_kafka_topic(topic_name)
+        self.kafka_topics.append(topic)
+        return topic_name
+
+    def create_kafka_conn_secret(self, topic):
+        """
+        Create secret from a JSON file that
+        defines the Kafka connection configuration
+
+        Args:
+            topic(str): Name of the Kafka topic
+
+        Returns:
+            secret_ocp_obj: OCP instance of the created secret
+            conn_file_name: Name of the JSON file
+        """
+        conn_name = create_unique_resource_name(
+            resource_description="nb-notif", resource_type="kafka-conn"
+        )
+        secret_name = conn_name + "-secret"
+        conn_file_name = ""
+
+        kafka_conn_config = {
+            "metadata.broker.list": "my-cluster-kafka-bootstrap.myproject.svc.cluster.local:9092",
+            "notification_protocol": "kafka",
+            "topic": topic,
+            "name": conn_name,
+        }
+
+        with tempfile.NamedTemporaryFile(
+            mode="w+", prefix="kafka_conn_", suffix=".json", delete=True
+        ) as conn_file:
+            conn_file_name = os.path.basename(conn_file.name)
+            conn_file.write(json.dumps(kafka_conn_config))
+            conn_file.flush()  # Ensure that the data is written
+
+            OCP().exec_oc_cmd(
+                f"create secret generic {secret_name} --from-file={conn_file.name} -n {self.namespace}"
+            )
+
+        secret_ocp_obj = OCP(
+            kind="secret",
+            namespace=self.namespace,
+            resource_name=secret_name,
+        )
+        self.conn_secrets.append(secret_ocp_obj)
+        return secret_ocp_obj, conn_file_name
+
+    def add_notif_conn_to_noobaa_cr(self, secret):
+        """
+        Add a connection secret to list of bucket notifications
+        connections in the NooBaa CR.
+
+        Args:
+            secret(ocs_ci.ocs.ocp.OCP): OCP instance of the secret to add
+        """
+        conn_data = {
+            "name": secret.resource_name,
+            "namespace": secret.namespace,
+        }
+        patch_path = "/spec/bucketNotifications/connections"
+        add_op = [{"op": "add", "path": f"{patch_path}/-", "value": conn_data}]
+        self.nb_config_resource.patch(
+            resource_name=constants.NOOBAA_RESOURCE_NAME,
+            params=json.dumps(add_op),
+            format_type="json",
+        )
+
+        nb_pods = [pod.name for pod in get_noobaa_endpoint_pods()]
+        nb_pods += [constants.NOOBAA_CORE_POD]
+        wait_for_pods_to_be_running(
+            namespace=self.namespace,
+            pod_names=nb_pods,
+            timeout=60,
+            sleep=10,
+        )
```

Comment:
Can we replace this with `get_noobaa_pods()` method from resources.pod.py?

Reply:
Fixed in https://github.com/red-hat-storage/ocs-ci/pull/11041/commits/532cf1636eb019e1906c95e8e390174639bcaf40